OAuth/ResourceOwner/AbstractResourceOwner.php

```diff
@@ -301,9 +301,9 @@ protected function generateNonce()
     /**
      * Configure the option resolver
      *
-     * @param OptionsResolverInterface $resolver
+     * @param OptionsResolver $resolver
      */
-    protected function configureOptions(OptionsResolverInterface $resolver)
+    protected function configureOptions(OptionsResolver $resolver)
```

Comment:
@Soullivaneuh Can you revert this change? It's BC break & it would require all code depending on it to be adjusted, I guess that we can't change this BC before `0.5`.